CODEOWNERS

```diff
@@ -8,12 +8,13 @@
 
 # Source
 /src/                                       @box/webapp
-/src/*                                      @box/preview @box/ui-elements
-/src/api/                                   @box/preview @box/ui-elements
+/src/*                                      @box/ui-elements
```

Comment:
not sure i understand the difference between these two sources. @coderabbitai can you help me understand?

Reply:
thanks for explaining rabbit

Reply:
thanks for explaining rabbit

Reply:
no, thanks for explaining @coderabbitai